ui/apps/dashboard/src/components/Billing/Addons/MetricsExportEntitlementValue.tsx

```diff
@@ -0,0 +1,49 @@
+import { entitlementSecondsToStr } from '@/utils/entitlementTimeFmt';
+
+type Props = {
+  metricsExportEnabled: boolean;
+  granularitySeconds?: number;
+  freshnessSeconds?: number;
+};
+
+export default function MetricsExportEntitlementValue({
+  metricsExportEnabled,
+  granularitySeconds,
+  freshnessSeconds,
+}: Props) {
+  if (!metricsExportEnabled) {
+    return 'Not enabled';
+  }
+
+  if (!granularitySeconds || !freshnessSeconds) {
+    throw new Error(
+      'granularitySeconds and freshnessSeconds must be given when metrics export is enabled'
+    );
+  }
+
+  return (
+    <>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          gap: '0.5rem',
+        }}
+      >
+        <span className="font-medium">Enabled</span>
+        <span
+          className="border-subtle"
+          style={{ borderLeftWidth: '1px', borderRightWidth: '1px' }}
+        ></span>
+        <span className="text-muted">Granularity</span>
+        <span className="font-medium">{entitlementSecondsToStr(granularitySeconds)}</span>
+        <span
+          className="border-subtle"
+          style={{ borderLeftWidth: '1px', borderRightWidth: '1px' }}
+        ></span>
+        <span className="text-muted">Freshness</span>
```

Comment:
Can we publicly call this "Delay" in the UI? I know I pushed for "freshness" during spec and development, but in writing the docs last week and comparing how other solutions talk about metrics freshness, I think "Delay" may be more clear to folks and easier to explain. 

NOTE - we'll keep the backend all the same for now and wait for customer feedback if "delay" makes sense or we need to be more explicit like "Metrics delay."

Reply:
@djfarrelly I'll open another PR with that change